packages/design-system/src/components/Accordion/AccordionComponents.tsx

```diff
@@ -38,19 +35,18 @@ const AccordionComponents = {
     return (
       <div
         className={classNames(
-          "text-accordion-fontDescription font-accordion-weightBody relative overflow-hidden pr-accordion-paddingBase text-accordion-base-icon transition-all duration-300 ease-in",
-          iconVariants[iconPosition]
+          "text-accordion-sizeBody font-accordion-body relative overflow-hidden text-accordion-base-icon transition-all duration-300 ease-in",
+          isIconOnTheLeft
+            ? "pl-accordion-bodyLIconVariantL"
+            : "pl-accordion-bodyRIconVariantL"
```

Comment:
```suggestion
          isIconOnTheLeft
            ? "pl-accordion-bodyLIconVariantL"
            : "pl-accordion-bodyRIconVariantR"
```
But I think it wasn't a necessary change. We are trying to keep the previous pattern in DS because it's more readable.

Reply:
ok, i'll change it back to an object